### ISSUE TITLE
Sample Labels input needed

### ISSUE THREAD

**Issue by EMCP** [cfotache/pytorch_objectdetecttrack#3]
https://github.com/cfotache/pytorch_objectdetecttrack/blob/master/utils/datasets.py#L92-L116

It would be helpful to include the label document so I can make sense of the code that goes inside the above area..  Not sure how to return the Labels.. seems I need to return a matrix of all classes?  Based on a list of pre-filled classes?  I want to train new classes with transfer learning and a dataset of images and bounding boxes..

currently my labels are a single text file for each image.. so I will open it and load the text file
```
MyExampleClass [859, 36, 2329, 1801]

```

And I seem to need to return a `torch.from_numpy` likeso

```
        if labels is not None:
            filled_labels[range(len(labels))[:self.max_objects]] = labels[:self.max_objects]
        filled_labels = torch.from_numpy(filled_labels)
```

is the `filled_labels` an array of `[[x1,y1,x2,y2,classification]....]` ?

**Comment by EMCP** [comment]
I see now,

it's a matrix in which the class is the final col.. with a 1 in place for whichever class this represents
